src/js/validation/has0x/validateHas0x.js

```diff
@@ -0,0 +1,6 @@
+var AMPParse = AMPParse || {};
+
+AMPParse.validateHas0x = function(hexString)
+{
+    return /^0[xX]/.test(hexString);
+}
```

Comment:
looks like it was in the design, not the requirements, so I'm fine with making 0x required, we can update the design to match